api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactory.java

```diff
@@ -95,17 +100,29 @@ protected Long calculateQueueTime(LocalDateTime timestamp) {
         customDimensions.put(WESTMINSTER_PARLIAMENTARY_CONSTITUENCY.getFieldName(), postcodeData.getParliamentaryConstituency());
         customDimensions.put(CLINICAL_COMMISSIONING_GROUP.getFieldName(), postcodeData.getCcg());
         customDimensions.put(CLINICAL_COMMISSIONING_GROUP_CODE.getFieldName(), postcodeData.getCodes().getCcg());
-        customDimensions.put(QUALIFYING_BENEFIT.getFieldName(), context.getIdentityAndEligibilityResponse().getQualifyingBenefits());
+        customDimensions.put(QUALIFYING_BENEFIT.getFieldName(), getQualifyingBenefits(identityAndEligibilityResponse));
 
         return customDimensions;
     }
 
+    private List<LocalDate> getDobOfChildrenUnder4(Claim claim, CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse) {
+        return identityAndEligibilityResponse == null
+                ? claim.getClaimant().getInitiallyDeclaredChildrenDob()
+                : identityAndEligibilityResponse.getDobOfChildrenUnder4();
```

Comment:
This is a change in behaviour here rather than fixing a bug, this is saying that if there aren't any children returned from the eligibility service then we trust what the Claimant has told us, is that what you intended? If so, this feels like it should be in its own story and documented in business rules somewhere rather than fixing a NPE bug.

Reply:
That's not quite correct, it's saying if there's no identityAndEligibilityResponse (due to not calling the eligibility service when the claim's a duplicate), we get the initially declared children. Before we just threw a NPE and retried the message.

Reply:
Yes, I can see that, that's what I was trying to say. Still seems like way more than fixing a NPE, if that's what was being done then it should just handle the null and return null but not change the logic to use the initially declared children.

Reply:
@YetAnotherMatt This is something I wanted to mention before you left. With the PR as is, you can't tell them difference in GA between a claim with children dobs from DWP or from the claimant.

Reply:
Given that it's only used for reporting - and if the identityAndEligibilityResponse is null we're reporting a rejected claim - I don't see it as an issue.